### PR TITLE
Add high temperature and short circuit Solar Charger alarms

### PR DESCRIPTION
**Pull request by DanielMcInnes**

### CODE REVIEW

File: pages/solar/SolarChargerPage.qml [hunk 1]
```diff
@@ -146,24 +147,43 @@ Page {
...
 					uid: root.solarCharger.serviceUid + "/Alarms/LowVoltage"
 				}
 				VeQuickItem {
 					id: highBatteryAlarm
+
+					//% "High battery voltage alarm"
+					property string text: qsTrId("charger_alarms_high_battery_voltage_alarm")
 					uid: root.solarCharger.serviceUid + "/Alarms/HighVoltage"
 				}
+				VeQuickItem {
```
Comment (DanielMcInnes) [inline pages/solar/SolarChargerPage.qml:173]: These alarm items could also live in SolarCharger if you think that's better

File: pages/solar/SolarChargerPage.qml [hunk 2]
```diff
@@ -146,24 +147,43 @@ Page {
...
+					property string text: qsTrId("charger_alarms_high_temperature_alarm")
+					uid: root.solarCharger.serviceUid + "/Alarms/HighTemperature"
+				}
+				VeQuickItem {
+					id: shortCircuitAlarm
+
+					//% "Short circuit alarm"
+					property string text: qsTrId("charger_alarms_short_circuit_alarm")
+					uid: root.solarCharger.serviceUid + "/Alarms/ShortCircuit"
+				}
```
Comment (jpetrell) [inline pages/solar/SolarChargerPage.qml:186]: gui-v2 side [change](https://github.com/victronenergy/gui/commit/49512faa53246f2a7c23192c8beaea26a1b7cccd) grouped/moved these alarm state items  to sub-page under "Alarm status"?

Comment (DanielMcInnes) [reply pages/solar/SolarChargerPage.qml:186]: At the moment they are under a sub-page "Alarms & Errors", would you prefer "Alarm Status"?

File: pages/solar/SolarChargerPage.qml (hunk 1 above)
Comment (blammit) [reply pages/solar/SolarChargerPage.qml:173]: No, better to keep it here if they aren't used outside of settings

File: pages/solar/SolarChargerPage.qml [hunk 3]
```diff
@@ -146,24 +147,43 @@ Page {
...
-							{ "title": text, "solarCharger": root.solarCharger })
+							{	"title": text,
+								"solarCharger": root.solarCharger })
 				}
 
 				VeQuickItem {
 					id: lowBatteryAlarm
+
+					//% "Low battery voltage alarm"
+					property string text: qsTrId("charger_alarms_low_battery_voltage_alarm")
```
Comment (blammit) [inline pages/solar/SolarChargerPage.qml:163]: Why does this VeQuickItem need a `text` property? (Same for the VeQuickItems below)

**Review by blammit — COMMENTED** [review]
Could you add the issue number to the commit message?

File: pages/solar/SolarChargerPage.qml [hunk 4]
```diff
@@ -146,24 +147,43 @@ Page {
...
-				// TODO update this binding to consider 'active alarms' section when it is
-				// implemented for the page.
-				enabled: lowBatteryAlarm.isValid || highBatteryAlarm.isValid
+				enabled: lowBatteryAlarm.isValid || highBatteryAlarm.isValid || highTemperatureAlarm.isValid || shortCircuitAlarm.isValid
 						 || root.solarCharger.errorModel.count
 
 				onClicked: {
 					Global.pageManager.pushPage("/pages/solar/SolarChargerAlarmsAndErrorsPage.qml",
-							{ "title": text, "solarCharger": root.solarCharger })
+							{	"title": text,
```
[the remark this answers is not in the document]
Comment (DanielMcInnes) [reply pages/solar/SolarChargerPage.qml]: reverted

File: pages/solar/SolarChargerPage.qml (hunk 3 above)
Comment (DanielMcInnes) [reply pages/solar/SolarChargerPage.qml]: reverted

File: pages/solar/SolarChargerPage.qml (hunk 2 above)
Comment (jpetrell) [reply pages/solar/SolarChargerPage.qml:173]: No that is perfect. Sorry I am just blind. :sweat_smile:

**Review by jpetrell — APPROVED** [review]

**Comment by DanielMcInnes** [comment]
> Could you add the issue number to the commit message?

DOne

**Review by blammit — APPROVED** [review]

**Pull request merged**